handbook/product/onboarding/index.md

```diff
@@ -1,72 +1,115 @@
 # Product team onboarding
 
-Welcome to Sourcegraph! As a member of the product team, it is your job to be the voice of the user, and to balance that with the goals of the company.
+Welcome to Sourcegraph! As a member of the product team, it is your job to be the voice of the user, and to balance that with the goals of the company. For all [roles on the product team](../roles/index.md), you will have a very similar first week. Weeks 2-4 will vary depending on your role and areas of focus, but the high level goals for each week will be the same. 
 
```

Comment:
Both great additions, but can we change the tone of the ‘no stupid questions’ section? I really love what SQS mentions in company meetings with no hires. Perhaps we can adopt that to make it work for your questions comment? Suggestion:

### Everyone is here to support you as you onboard

Joining a new company can be overwhelming — there’s a lot to learn! As you navigate your first few weeks at Sourcegraph, we want you to know that everyone on the team is here to help and will gladly drop anything to answer any question you have.

Reply:
Rather than making these part of the product onboarding docs, they are relevant to everyone and should be added to general onboarding. I'll make the update!